app/monitoring/Metrics.scala

```diff
@@ -0,0 +1,11 @@
+package monitoring
+
+import com.amazonaws.regions.{Region, Regions}
+import com.gu.monitoring.CloudWatch
+import configuration.Config
+
+trait Metrics extends CloudWatch {
+  val region = Region.getRegion(Regions.EU_WEST_1)
+  val stage = Config.stage
+  val application = "SubscriptionsFrontend"
```

Comment:
we have an application constant that holds the application name, I think it's better to keep it unified https://github.com/guardian/subscriptions-frontend/blob/bbbba800417d8d2c2a0648f23898635733d9efce/app/configuration/Config.scala#L13